ct/bookstack.sh

```diff
@@ -70,9 +70,10 @@ if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_v
   unzip -q v${RELEASE}.zip
   mv BookStack-${RELEASE} /opt/bookstack
   mv /opt/.env /opt/bookstack/.env
-  COMPOSER_ALLOW_SUPERUSER=1 composer install --no-dev  &>/dev/null
-  php artisan key:generate &>/dev/null
-  php artisan migrate &>/dev/null
+  cd /opt/bookstack
+  composer install --no-dev --no-interaction &>/dev/null
```

Comment:
Why remove `COMPOSER_ALLOW_SUPERUSER=1`? What was the error message before?

Reply:
COMPOSER_ALLOW_SUPERUSER=1 is not needed with **--no-interaction**

If failed at this `COMPOSER_ALLOW_SUPERUSER=1 composer install --no-dev &> /dev/null`

In my opinion, the update was never working correctly as it was set up bevor.